CAIPs/caip-261.md

```diff
@@ -0,0 +1,466 @@
+
+## Abstract
+
+Webs of trust form peer-to-peer trust graphs through issuance of trust and/or distrust assertions among peers, offering the possibility to calculate valuable insights such as reputation scores by using graph theory.
+
+This proposal outlines a comprehensive data framework designed to facilitate the representation, management, verification and consumption of explicit trust assertions for peers within a decentralized environment.
+
+The proposed data framework is designed to be flexible, extensible, and scalable, ensuring compatibility across any ecosystem and facilitating a wide range of trust-based use-cases.
+
+By establishing a common language for peer trust assertions, this specification enables diverse ecosystems to interoperate with a shared understanding of trust, benefiting from each other and creating synergies that strengthen insights generated from webs of trust.
+
+## Specification
+
+This specification outlines methods for representing peer trust assertions, managing their lifecycle, and verifying their authenticity.
+It provides as well guidelines for representing insights calculated from a web of trust.
+The application of graph theory to utilize these webs of trust is beyond the scope of this proposal, but some examples are provided.
+
+### Trust Representation
+
+In the following diagram, we depict the process where an issuer of a long-lived and verifiable document formally asserts their trust in a subject, highlighting the foundational relationship within a peer-to-peer trust-based system.
+
+![diagram1](https://github.com/dayksx/CAIPs/assets/77788154/e14b4bcb-afc2-4578-9e33-1c0c8bb927e2)
+
+#### Identifier Scheme
+
+##### Peers Identification
+
+Trust assertions are made about, and by, peers identifiable with identifiers derived from public keys or directly using a public key. This enables peers to sign assertions and be verified.
+
+Identification of peers SHOULD make use of key pair-based mechanism to facilitate interoperability and verifiability.
+- **[Decentralized Identifiers (DIDs)][DID]** with key pair-based methods for off-chain assertions.
+  - **did:pkh (Public Key Hash):** This method is designed to be chain-agnostic, enabling the creation of DIDs based on public key hashes from a variety of blockchains. Its universality promotes interoperability and simplifies identity management across diverse ecosystems. This approach is particularly useful for applications that aim to operate across multiple cryptographically-secured networks (like blockchains) without being tied to any specific one.
+  - **did:key:** This method allows for the generation of DIDs directly from cryptographic public keys, supporting various key types such as ECDSA, EdDSA, Schnorr Signatures, and RSA. It is straightforward and self-contained, with no need for interaction with blockchain networks. This makes it fast and cost-effective for scenarios that do not require decentralized verification.

```

Comment:
```suggestion
  - **did:key:** This method allows for the generation of DIDs directly from cryptographic public keys, supporting various key types such as ECDSA, EdDSA, Schnorr Signatures, and RSA. It is straightforward and self-contained, with no need for interaction with blockchain networks. This makes it fast and cost-effective for scenarios that do not require on-chain verification or cross-chain use-cases using keytypes not available in the target verifier's toolchain.
```

UCANs use `did:key`s for decentralized verification! The question of whether or not verification is decentralized is orthogonal to how the key is expressed; best to get more precise about who needs to use which did expression where, whether my suggestion works or not.